docs/api/ipc-main.md

```diff
@@ -90,8 +90,14 @@ Set this to the value to be returned in a synchronous message.
 
 ### `event.sender`
 
-Returns the `webContents` that sent the message, you can call
+Returns the `ipcRenderer` that sent the message, you can call
```

Comment:
This isn't a change, this is accurately describing the existing behavior. `event.sender` isn't a WebContents, it only has `send` and friends